server/webpa.go

```diff
@@ -76,10 +81,29 @@ func (b *Basic) New(logger log.Logger, handler http.Handler) *http.Server {
 		return nil
 	}
 
+	var tlsConfig *tls.Config
+	certificateFile, keyFile := b.Certificate()
+	if len(certificateFile) > 0 && len(keyFile) > 0 && len(b.ClientCACertFile) > 0 {
```

Comment:
Theae conditions should be separate.  As written, this code requires MTLS to do HTTPS.  We should be able to configure an HTTPS server with optional MTLS.

Reply:
Oh I see ... this is only MTLS.  That's fine then.  Disregard.